### DISCUSSION TITLE
My Character wont work no matter What

### DISCUSSION THREAD

**Discussion by Megaman9009** [KadeArchive/Kade-Engine#575] [Q&A]
![GRRRR](https://user-images.githubusercontent.com/84752715/119435685-aa44c400-bce8-11eb-9d12-50fe56e5581a.PNG)
![GR](https://user-images.githubusercontent.com/84752715/119422160-6e9d0080-bcce-11eb-9549-1ea5a1dece7c.PNG)
![GRRR](https://user-images.githubusercontent.com/84752715/119422162-6fce2d80-bcce-11eb-9644-4c7a3452b12e.PNG)
![GRR](https://user-images.githubusercontent.com/84752715/119422163-6fce2d80-bcce-11eb-86b4-3578afcdf7a7.PNG)

It wont work

**Comment by guizinhomartinez** [comment]
you probally forgot to put the character into preload/data characters file

**Reply by Megaman9009** [reply]
I did

**Reply by Megaman9009** [reply]
I put it in character list already

**Comment by Megaman9009** [comment]
I did its on the screenshot

**Reply by guizinhomartinez** [reply]
no its not the only thing that appears is the xml of your character

**Reply by Megaman9009** [reply]
What do you mean?

**Reply by guizinhomartinez** [reply]
the notepad screenshot is probablly the file keith.xml

**Reply by Megaman9009** [reply]
It is

**Comment by pepperly-art** [comment]
Did you add keith to characterlist.txt ?

![image](https://user-images.githubusercontent.com/82689031/119434921-29d19380-bce7-11eb-828e-86ff438b6253.png)

**Reply by Megaman9009** [reply]
Yeah

**Reply by Megaman9009** [reply]
I swear  I did EVERYTHING

**Comment by Megaman9009** [comment]
I downloaded another kade engine and did it again. It works this time but I still don't understand why this one doesn't

**Reply by guizinhomartinez** [reply]
bro send a link of download of your character and i code it in

**Reply by Megaman9009** [reply]
its fine thanks though

**Comment by prockscac** [comment]
https://youtu.be/ddzZ8hJROWw this will probably work cause it worked for me.
